packages/nimble-components/src/rich-text-editor/index.ts

```diff
@@ -347,6 +355,18 @@ export class RichTextEditor extends FoundationElement {
                 return false;
         }
     }
+
+    // In Firefox browser, Once the editor gets focused, the blinking caret will be visible until we click format buttons (Bold, Italic ...) in the Firefox browser (Changing focus).
+    // But once the any of the buttons clicked, the editor internally has its focus but the blinking caret disappears.
+    // As a workaround, manually triggering blur and setting focus on editor will hepls making the caret re-appears.
+    // It seems like this issue in not fixed in firefox browser yet, Created Issue https://github.com/ni/nimble/issues/1454 tracks removal of this workaround.
```

Comment:
I don't think the created issue tracks removal of this workaround. Did you meant to say fix? 

However consider removing this redundant information, as it's already in PR description. 

```suggestion
    // It seems like this issue in not fixed in firefox browser yet, Created Issue https://github.com/ni/nimble/issues/1454 tracks removal of this workaround.
```